### ISSUE TITLE
HTTP API very slow

### ISSUE THREAD

**Issue by mcweba** [swisspost/vertx-redisques#213]
The `/queuing/monitor` and `/queuing/statistics` HTTP API endpoints in newer vertx-redisques versions are really slow. The other endpoints like `/queuing/queues` or `/queuing/locks` are fast in the newer versions too. Check the two compared versions below:

## 3.0.31
```bash
curl -o /dev/null  -s -w "Response Time: %{time_total}s\n"  http://localhost:7018/queuing/monitor
Response Time: 0.006s
```

## 4.1.2
```bash
curl -o /dev/null  -s -w "Response Time: %{time_total}s\n"  http://localhost:7018/queuing/monitor
Response Time: 0.780s
```

This is around 130 times slower in the new version. One thing related could be that newer versions have more output. See:

```json
{
    "queues": [
        {
            "name": "nemo-to-dormakaba-v2",
            "size": 151,
            "lastDequeueAttempt": "",
            "lastDequeueSuccess": "",
            "nextDequeueDueTimestamp": ""
        },
        {
            "name": "edds-delivery",
            "size": 50,
            "lastDequeueAttempt": "",
            "lastDequeueSuccess": "",
            "nextDequeueDueTimestamp": ""
        },
        {
            "name": "edds-deliveryEvent-kafka-fwd",
            "size": 17,
            "lastDequeueAttempt": "",
            "lastDequeueSuccess": "",
            "nextDequeueDueTimestamp": ""
        }
    ]
}
```
The output of the older versions is:
```json
{
    "queues": [
        {
            "name": "nemo-to-dormakaba-v2",
            "size": 3010
        },
        {
            "name": "edds-delivery",
            "size": 111
        },
        {
            "name": "edds-sisy-signatureEvent-kafka-fwd",
            "size": 59
        }
    ]
}
```

**Comment by mcweba** [comment]
QueueStatistics can be disabled by configuration. In this case I would also expect the API to not collect/process this data. Therefore the response times should be as fast as before

**Comment by mcweba** [comment]
Fixed in [v4.1.3](https://github.com/swisspost/vertx-redisques/releases/tag/v4.1.3)
